KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -251,20 +267,6 @@ private void draw(Graphics g) {
             }
         }
 
-        //draw testing art
-        //cards
-        g2d.drawImage(CARD_CLAY, 100, 1080 - 125, null); //space them 100 pixels apart and align the hight to 2 from the bottom
-        g2d.drawImage(CARD_ORE, 200, 1080 - 125, null);
-        g2d.drawImage(CARD_WHEAT, 300, 1080 - 125, null);
-        g2d.drawImage(CARD_WOOD, 400, 1080 - 125, null);
-        g2d.drawImage(CARD_SHEEP, 500, 1080 - 125, null);
```

Comment:
Hey, why kill the cards?

Reply:
oh sorry i meant to delete the settlement test

Reply:
Those will be added back in the form of drawing the player's hand/deck anyway

Reply:
fair enough